spacemacs/tests/tests.el

```diff
@@ -0,0 +1,2 @@
+(ert-deftest a-random-test ()
```

Comment:
Is a `require` for the test `init.el` file needed here ?
I don't see the test `init.el` file loaded anywhere.